src/useCases/drive/deleteShareLink.ts

```diff
@@ -19,7 +18,7 @@ export const deleteShareLink = async ({ shareId, type }: { shareId: string; type
     analytics.track(DriveAnalyticsEvent.SharedLinkDeleted, {
       type,
     });
-    return result;
+    return true;
```

Comment:
you sure about this? are we going to assume that whatever 200 the backend returns (or simply not an error) is a successful operation?

Reply:
What other case apart from 200 are we going to want to handle here? If it is unsuccessful it will give some other HTTP error code and show the user that it has not been deleted. 
Is there any other success case we want to handle in the frontend?

Reply:
a 400 status code is a good example of that, or maybe even a 204